### ISSUE TITLE
Is there any way to do incremental training on CCNF & ce-clm models?

### ISSUE THREAD

**Issue by EDMIG** [TadasBaltrusaitis/OpenFace#497]
Thx!

**Comment by A2Zadeh** [comment]
Yes. The training code for CE-CLM allows you to continue the training if you give it a model. Say for example you train a model for 100 epochs and now you want to do 500 you simply feed the best of the first 100 to the script and ask the script to continue for 400 more.
